docs/source/ibis.md

```diff
@@ -422,55 +422,6 @@ validated_t = t.pipe(ModelWithDFChecks.validate)
 print(validated_t)
 ```
 
-## Data-level Validation with LazyFrames
-
-As mentioned earlier in this page, by default calling `schema.validate` on
-a `pl.LazyFrame` will only perform schema-level validation checks. If you want
-to validate data-level properties on a `pl.LazyFrame`, the recommended way
-would be to first call `.collect()`:
-
-```{code-cell} python
-:tags: ["raises-exception"]
-class SimpleModel(pa.DataFrameModel):
-        a: int
-
-lf: pl.LazyFrame = (
-    pl.LazyFrame({"a": [1.0, 2.0, 3.0]})
-    .cast({"a": pl.Int64})
-    .collect()  # convert to pl.DataFrame
-    .pipe(SimpleModel.validate)
-    .lazy()     # convert back to pl.LazyFrame
-    # do more lazy operations
-)
-```
-
-This syntax is nice because it's clear what's happening just from reading the
-code. Pandera schemas serve as a clear point in the method chain where the data
-is materialized.
-
-However, if you don't mind a little magic 🪄, you can set the
-`PANDERA_VALIDATION_DEPTH` environment variable to `SCHEMA_AND_DATA` to
```

Comment:
@cosmicBboy I wonder if removing this means can also remove the validation scope decorators from the implementation? I don't think it's necessary here, since we do the data checks along with the schema checks.

Reply:
let's preserve the validation scope decorators, it's used for error reporting as well.

Reply:
Sounds good; I won't remove the decorators from the code, but I'm still removing this section, since data-level checks are already handled by default.